resources/services/cloudwatchlogs/log_groups_mock_test.go

```diff
@@ -0,0 +1,32 @@
+package cloudwatchlogs
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs/types"
+	"github.com/bxcodec/faker"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-aws/client/mocks"
+	"github.com/golang/mock/gomock"
+)
+
+func buildCloudwatchLogsLogGroupsMock(t *testing.T, ctrl *gomock.Controller) client.Services {
+	m := mocks.NewMockCloudwatchLogsClient(ctrl)
+	l := types.LogGroup{}
+	err := faker.FakeData(&l)
+	if err != nil {
+		t.Fatal(err)
+	}
+	m.EXPECT().DescribeLogGroups(gomock.Any(), gomock.Any(), gomock.Any()).Return(
+		&cloudwatchlogs.DescribeLogGroupsOutput{
+			LogGroups: []types.LogGroup{l},
+		}, nil)
+	return client.Services{
+		CloudwatchLogs: m,
+	}
+}
+
+func TestCloudwatchlogsLogGroups(t *testing.T) {
```

Comment:
```
--- FAIL: TestCloudwatchlogsLogGroups (0.04s)
    provider.go:219: [INFO] calculated max goroutines for fetch execution [max_goroutines 65536]
    log_groups.go:111: Unexpected call to *mocks.MockCloudwatchLogsClient.ListTagsLogGroup([context.Background.WithCancel 0xc000596580]) at /home/runner/work/cq-provider-aws/cq-provider-aws/resources/services/cloudwatchlogs/log_groups.go:111 because: there are no expected calls of the method "ListTagsLogGroup" for that receiver
    --- FAIL: TestCloudwatchlogsLogGroups/test_resource (0.04s)
        resource.go:92: fetch resource test_resource
        --- FAIL: TestCloudwatchlogsLogGroups/test_resource/aws_cloudwatchlogs_log_groups (0.00s)
            resource.go:92: expected to have at least 1 entry at table aws_cloudwatchlogs_log_groups got zero
```

You need to address this before the PR may be merged.

Reply:
Added, thank you!